src/App.jsx

```diff
@@ -14,33 +16,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_BY_ALPHABET = 'Sort alphabetically';
+const SORT_BY_LENGTH = 'Sort by length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [sort, setSort] = useState({ type: '', reversed: false });
```

Comment:
```suggestion
  const [sortParams, setSortParams] = useState({ type: '', isReversed: false });
```
Boolean properties should start from `is` or `has`